core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -329,9 +329,13 @@ public int getValue() {
   /**
    * Gets the block size of a file in under file system, in bytes.
    *
+   * @deprecated block size should be returned as part of the {@link UfsFileStatus} from
```

Comment:
not sure if you want to deprecate this API because other two APIs providing structures covering this piece of info?
e.g., in an empty UFS, what is the expected approach to get block size then?

Reply:
My thoughts on why I want to deprecate this method -
This method is only useful if the cost of getting the block size is much less than getting the file status (this is true for certain UFSes, such as object stores) AND we use blocksize by itself often in our code (this is not true, we only use block size in one place and it is alongside getFileStatus).
Going forward, I don't think we will have a use case for only fetching the block size of a file from the UFS without fetching the rest of the file status, which is why I think we can deprecate this method.
When you say empty ufs, do you mean a ufs with no files? Block size is tied to a file (ie. it takes the file path as an argument).